app-doc/doxygen/doxygen-1.8.12.ebuild

```diff
@@ -7,17 +7,17 @@ PYTHON_COMPAT=( python{2_7,3_4,3_5} )
 inherit cmake-utils eutils fdo-mime flag-o-matic python-any-r1
 if [[ ${PV} = *9999* ]]; then
 	inherit git-r3
-	EGIT_REPO_URI="git://github.com/doxygen/doxygen.git"
+	EGIT_REPO_URI="https://github.com/doxygen/doxygen.git"
 	SRC_URI=""
 	KEYWORDS="alpha arm hppa ia64 sparc"
 else
-	SRC_URI="http://ftp.stack.nl/pub/users/dimitri/${P}.src.tar.gz"
+	SRC_URI="https://ftp.stack.nl/pub/users/dimitri/${P}.src.tar.gz"
```

Comment:
https://ftp.stack.nl/pub/users/dimitri/doxygen-1.8.12.src.tar.gz works for me with Firefox 55.0b12. The directory https://ftp.stack.nl/pub/users/dimitri/ returns a 404 though, instead of the correct error code for an access denied error.